src/main/java/Login.java

```diff
@@ -75,11 +75,20 @@ protected void doPost (HttpServletRequest req,
 
         // Redirect to another page
         // req.getRequestDispatcher("./HelloServlet").forward(req, res);
-        res.sendRedirect("/servlet-1.0");
+        res.sendRedirect("/risto89-1.0/profile");
     }
     else {
-       // TODO: Redirect to error page
+    	res.sendRedirect("/risto89-1.0/login");
     }
 
   }
+
+  protected void doGet (HttpServletRequest req,
+  		     HttpServletResponse res)
+     throws ServletException, IOException
+  {
+    RequestDispatcher dispatcher = req.getRequestDispatcher("/Loginpage.html");
```

Comment:
You can do this in one line
`req.getRequestDispatcher("/Loginpage.html").include(req, res);`